package.json

```diff
@@ -58,7 +58,7 @@
   },
   "devDependencies": {
     "@node-rs/jsonwebtoken": "^0.5.9",
-    "@sinonjs/fake-timers": "^13.0.5",
+    "@sinonjs/fake-timers": "^14.0.0",
```

Comment:
any chance we can use Node's built in mock timers?

Reply:
The last time I tried it was not working well on node 20, let me try that again